### PR TITLE
Nicsaal/debug activity

### PR DESCRIPTION
**Pull request by nicsaal**
Debug activity and all cleanup changes. There so much going on here I'm starting to get lost especially around the new map api

### CODE REVIEW

**Comment by oliversaal** [comment]
Need to remember to avoid the "Nicsaal/..." branch name style titles - use a friendly high-level title to describe the feature work. Example: "Add new activity view to display raw data; simplified main activity to leverage GoogleMap class"

File: WITW/app/src/main/java/com/digitalartthingy/witw/MainActivity.java [hunk 1]
```diff
@@ -201,187 +153,84 @@ public boolean onOptionsItemSelected(MenuItem item) {
...
+            //
+            // The following camera methods have been deprecated - we need to find alternates
+            //
+
+            // Remember the zoom level if the camera position is changed
+            mGoogleMap.setOnCameraChangeListener(new GoogleMap.OnCameraChangeListener() {
+                @Override
+                public void onCameraChange(CameraPosition cameraPosition) {
+                    // TODO: Once we move to higher minSDK then we can use setMaxZoomPreference instead
+                    if (cameraPosition.zoom <= MAX_ZOOM_LEVEL_PREFERENCE) {
```
Comment (oliversaal) [inline WITW/app/src/main/java/com/digitalartthingy/witw/MainActivity.java]: Maybe we should consider changing this to "!=". That way the last zoom setting will be honored even if the myLocation or the user has changed it.

File: WITW/app/src/main/res/values/dimens.xml [hunk 2]
```diff
@@ -1,12 +1,7 @@
-<?xml version="1.0" encoding="utf-8"?>
-<!--
+<?xml version="1.0" encoding="utf-8"?><!--
```
Comment (oliversaal) [inline WITW/app/src/main/res/values/dimens.xml]: Revert this line as well.

File: WITW/app/src/main/res/values/strings.xml [hunk 3]
```diff
@@ -1,5 +1,4 @@
-<?xml version="1.0" encoding="utf-8"?>
-<!--
+<?xml version="1.0" encoding="utf-8"?><!--
```
Comment (oliversaal) [inline WITW/app/src/main/res/values/strings.xml]: Ditto

File: WITW/app/src/main/res/values/styles.xml [hunk 4]
```diff
@@ -1,13 +1,15 @@
-<?xml version="1.0" encoding="utf-8"?>
-<!--
+<?xml version="1.0" encoding="utf-8"?><!--
```
Comment (oliversaal) [inline WITW/app/src/main/res/values/styles.xml]: One more time

**Comment by oliversaal** [comment]
Be sure to "SQUASH and MERGE" when submitting - we only need a single commit in the master branch

File: WITW/app/src/main/AndroidManifest.xml [hunk 5]
```diff
@@ -1,5 +1,4 @@
-<?xml version="1.0" encoding="utf-8"?>
-<!--
+<?xml version="1.0" encoding="utf-8"?><!--
```
Comment (nicsaal) [inline WITW/app/src/main/AndroidManifest.xml]: done

File: WITW/app/src/main/res/values/dimens.xml (hunk 2 above)
Comment (nicsaal) [inline WITW/app/src/main/res/values/dimens.xml]: done

File: WITW/app/src/main/res/values/strings.xml (hunk 3 above)
Comment (nicsaal) [inline WITW/app/src/main/res/values/strings.xml]: done

File: WITW/app/src/main/res/values/styles.xml (hunk 4 above)
Comment (nicsaal) [inline WITW/app/src/main/res/values/styles.xml]: done

File: WITW/app/src/main/java/com/digitalartthingy/witw/MainActivity.java (hunk 1 above)
Comment (nicsaal) [inline WITW/app/src/main/java/com/digitalartthingy/witw/MainActivity.java]: I've changed this but I don't see how mZoom is being used in our app

Comment (nicsaal) [inline WITW/app/src/main/java/com/digitalartthingy/witw/MainActivity.java]: done

**Pull request merged**